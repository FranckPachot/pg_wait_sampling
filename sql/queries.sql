CREATE EXTENSION pg_wait_sampling;

WITH t as (SELECT sum(0) FROM pg_wait_sampling_current)
	SELECT sum(0) FROM generate_series(1, 2), t;

WITH t as (SELECT sum(0) FROM pg_wait_sampling_history)
	SELECT sum(0) FROM generate_series(1, 2), t;

WITH t as (SELECT sum(0) FROM pg_wait_sampling_profile)
	SELECT sum(0) FROM generate_series(1, 2), t;

SELECT pg_wait_sampling_reset_profile();

DROP EXTENSION pg_wait_sampling;
