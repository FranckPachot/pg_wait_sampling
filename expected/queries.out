CREATE EXTENSION pg_wait_sampling;
WITH t as (SELECT sum(0) FROM pg_wait_sampling_current)
	SELECT sum(0) FROM generate_series(1, 2), t;
 sum 
-----
   0
(1 row)

WITH t as (SELECT sum(0) FROM pg_wait_sampling_history)
	SELECT sum(0) FROM generate_series(1, 2), t;
 sum 
-----
   0
(1 row)

WITH t as (SELECT sum(0) FROM pg_wait_sampling_profile)
	SELECT sum(0) FROM generate_series(1, 2), t;
 sum 
-----
   0
(1 row)

SELECT pg_wait_sampling_reset_profile();
 pg_wait_sampling_reset_profile 
--------------------------------
 
(1 row)

DROP EXTENSION pg_wait_sampling;
